opentech/apply/funds/models.py

```diff
@@ -67,3 +96,73 @@ class ApplicationForm(models.Model):
 
     def __str__(self):
         return self.name
+
+
+class Round(AbstractStreamForm):
+    parent_page_types = ['funds.FundType']
+    subpage_types = []  # type: ignore
+
+    start_date = models.DateField(default=date.today)
+    end_date = models.DateField(
+        blank=True,
+        null=True,
+        default=date.today,
+        help_text='If no end date is provided the round will remain open.'
+    )
+
+    content_panels = AbstractStreamForm.content_panels + [
+        MultiFieldPanel([
+            FieldRowPanel([
+                FieldPanel('start_date'),
+                FieldPanel('end_date'),
+            ]),
+        ], heading="Dates")
+    ]
+
+    def get_defined_fields(self):
+        # Only return the first form, will need updating for when working with 2 stage WF
```

Comment:
Make note that it does not handle changing forms per round. i.e. it inherits the form from the fund type.

Reply:
I'm not too worried about this. The following tickets will look at start storing form data on the round object and this will be totally reworked.